alpine/Dockerfile

```
@@ -42,18 +42,17 @@ RUN apk add --no-cache \
 ENV PATH="/opt/jdk-${JAVA_VERSION}/bin:${PATH}"
 
 RUN case "$(jlink --version 2>&1)" in \
-      "17."*) set -- "--compress=2" ;; \
-      # the compression argument is different for JDK21
-      "21."*) set -- "--compress=zip-6" ;; \
-      *) echo "ERROR: unmanaged jlink version pattern" && exit 1 ;; \
-    esac; \
-    jlink \
-      --strip-java-debug-attributes \
-      "$1" \
-      --add-modules ALL-MODULE-PATH \
-      --no-man-pages \
-      --no-header-files \
-      --output /javaruntime
+    "17."*) options="--compress=2 --add-modules ALL-MODULE-PATH" ;; \
+    "21."*) options="--compress=zip-6 --add-modules ALL-MODULE-PATH" ;; \
+    "25"*) options="--compress=zip-6 --add-modules java.base,java.logging,java.xml,java.se" ;; \
+    *) echo "Unsupported Java version ($(jlink --version 2>&1))" && exit 1 ;; \
+esac && \
+jlink \
+    --strip-java-debug-attributes \
+    ${options} \
+    --no-man-pages \
+    --no-header-files \
+    --output /javaruntime
```

```suggestion
      "17."*) options="--compress=2 --add-modules ALL-MODULE-PATH" ;; \
      "21."*) options="--compress=zip-6 --add-modules ALL-MODULE-PATH" ;; \
      "25"*) options="--compress=zip-6 --add-modules java.base,java.logging,java.xml,java.se" ;; \
      *) echo "Unsupported Java version ($(jlink --version 2>&1))" && exit 1 ;; \
    esac && \
    jlink \
      --strip-java-debug-attributes \
      ${options} \
      --no-man-pages \
      --no-header-files \
      --output /javaruntime
```
request: restore original indentation

question: there is no dot after "25" like the other cases, is it expected?